src/pages/workspace/companyCards/addNew/CardInstructionsStep.tsx

```diff
@@ -34,6 +46,9 @@ function CardInstructionsStep({policyID}: CardInstructionsStepProps) {
     const isStripeFeedProvider = feedProvider === CONST.COMPANY_CARD.FEED_BANK_NAME.STRIPE;
     const isAmexFeedProvider = feedProvider === CONST.COMPANY_CARD.FEED_BANK_NAME.AMEX;
     const isOtherBankSelected = bank === CONST.COMPANY_CARDS.BANKS.OTHER;
+    const isVisaFeedProvider = feedProvider === CONST.COMPANY_CARD.FEED_BANK_NAME.VISA;
+    const isMastedCardFeedProvider = feedProvider === CONST.COMPANY_CARD.FEED_BANK_NAME.MASTER_CARD;
+    const translationKey = getCardInstructionHeader(isVisaFeedProvider, isMastedCardFeedProvider);
```

Comment:
Can we just pass the feedProvider here and handle the logic in the method instead of adding two boolean values here?